assets/js/base/components/button/style.scss

```diff
@@ -54,6 +54,12 @@
 			opacity: 1;
 		}
 	}
+
+	&.contained {
+		&:hover {
+			opacity: 0.9;
```

Comment:
We're not changing the opacity for Mini Cart and Add to Cart buttons.
I think we should remove this and keep it consistent with core button blocks. 

Slack: p1680026511680059/1680024656.322559-slack-C02UBB1EPEF